tooling/nargo_fmt/tests/expected/comment.nr

```diff
@@ -0,0 +1,14 @@
+fn comment1() {
+//
+}
+
+// random comment
+
+fn comment2() {
+// Test
+}
+fn comment3() {}
+fn comment4() {}
+fn comment5() {}
+fn comment6() {}
+fn comment7() {}
```

Comment:
A `fmt` command as a whole is greatly appreciated, but I'm going to block this PR while comments are being deleted. I think a `fmt` command that deletes comments would be a deal breaker for most users and don't want them using the command and accidentally deleting comments in the meantime while we work on it.

Preserving comments can be difficult in general though so we'd need some work in adding them to our syntax tree.